doc/src/tutorial/beam_problems.rst

```diff
@@ -0,0 +1,201 @@
+===========================================================
+Solving Beam Bending Problems using Discontinuity Functions
+===========================================================
+
+To make this document easier to read, we are going to enable pretty printing.
+
+    >>> from sympy import *
+    >>> x, y, z = symbols('x y z')
+    >>> init_printing(use_unicode=True, wrap_line=False)
+
+Beam
+====
+
+A Beam is a structural element that is capable of withstanding load
+primarily by resisting against bending. Beams are characterized by
+their second moment of area, their length and their elastic modulus.
+
+In Sympy, we can constuct a 2D beam objects with the following properties :
+
+- Length
+- Elastic Modulus
+- Second Moment of Area
+- Variable : A symbol that can be used as a variable along the length. By default,
+  this is set to ``Symbol(x)``.
+- Boundary Conditions
+    - bc_moment : Boundary conditions for moment.
+    - bc_slope : Boundary conditions for slope.
+    - bc_deflection : Boundary conditions for deflection.
+- Load Distribution
+
+We have following methods under the beam class:
+
+- apply_load
+- evaluate_reaction_forces
+- shear_force
+- bending_moment
+- slope
+- deflection
+
+
+Examples
+========
+
+Let us solve some beam bending problems using this module :
+
+**Example 1**
+
+A beam of length 9 meters is having a fixed support at the start.
+A distributed constant load of 8 kN/m is applied downward from the starting
+point till 5 meters away from the start. A clockwise moment of 50 kN.m is
+applied at 5 meters away from the start of the beam. A downward point load
+of 12 kN is applied at the end.
```

Comment:
Yah, I also felt so while documenting example problems. But how do we add sketches in the .rst file?